website/content/en/docs/setup/installation/operating-systems/nixos.md

```diff
@@ -1,14 +1,24 @@
 ---
 title: Install Vector on NixOS
 short: NixOS
-supported_installers: ["Nix", "Vector installer", "Docker"]
 weight: 6
 ---
 
-[Debian], also known as Debian GNU/Linux, is a Linux distribution composed of free and open-source software, developed by the community-supported Debian Project. This page covers installing and managing Vector on the Debian operating system.
+[NixOS] is a Linux distribution built on top of the Nix package manager. This
+page covers installing and managing Vector on NixOS.
 
-## Supported installers
```

Comment:
Could we leave the `Supported installers` section as-is? That would leave it consistent with the other OS pages under https://vector.dev/docs/setup/installation/operating-systems